src/assets/content.tsx

```diff
@@ -10,6 +10,18 @@ export const HOME_HEADER = "Let's get back to those trees, why don't we?";
 export const LANDING_TITLE = "Boston's Street Trees";
 export const LANDING_BODY =
   "Boston's trees do a lot for our community - they clean our polluted air, offer shade on scorching summer days and warmth on freezing winter nights, and prevent flooding during storms. At Speak for the Trees, we're determined to improve the size and health of our urban forest, especially in under-served and under-canopied communities.";
+export const ADOPTION_DIRECTIONS_HEADER = 'How to Adopt a Tree';
+export const FIND_DIRECTION =
+  'Find a tree near you by entering your address in the search bar at the at the top left of the map.';
+export const ICONS_DIRECTION =
+  'Click on the icons of trees you might be interested in adopting.';
+export const ADOPTED_TREE_ICON_DESCRIPTION = 'have already been adopted';
+export const TREE_ICON_DESCRIPTION =
+  'need less help but would still love to be adopted';
+export const YOUNG_TREE_ICON_DESCRIPTION =
+  'are young trees in the most in need of adopting';
```

Comment:
⛏️ This phrasing is a bit confusing

Reply:
Tried to make it a bit clearer, let me know what you think. We'll also double-check this with SFTT before the release.